ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -563,13 +564,8 @@ Import New Custom Image
 
 Open Custom Image Import Popup
     [Documentation]    Opens the Custom Image import view, using the appropriate button
-    ${first_image} =  Run Keyword And Return Status  Page Should Contain Element  xpath://button[.="Import image"]
-    IF  ${first_image}==True
-        Click Element  xpath://button[.="Import image"]
-    ELSE
-        Click Element  xpath://button[.="Import new image"]
-    END
-    Wait Until Page Contains    Import notebook images
+    Click Element  xpath://button[.="Import new image"]
+    Wait Until Page Contains    Import notebook image
```

Comment:
Are you sure in both cases (first image being imported and additional ones) the button says "import new image"?

Reply:
yes, I am sure; and the test is passing too :slightly_smiling_face: